wamr-compiler/CMakeLists.txt

```diff
@@ -387,6 +387,12 @@ if (NOT MSVC)
       target_link_libraries (wamrc ssp.a ws2_32)
   else()
       target_link_libraries (wamrc -ldl)
+      # Link libc++ statically to reduce the runtime dependency
+      target_link_libraries (wamrc -static-libstdc++)
+      # If not on macOS, link libgcc statically
+      if (NOT APPLE)
+        target_link_libraries (wamrc -static-libgcc)
+      endif()
```

Comment:
which instance of these libraries are used by shared libraries loaded by `--native-lib`?
is it ok to have multiple copies of these libraries in a process?

Reply:
I'm not sure which copy will be used in the native lib, but it seems OK since I have used a cxx based custom compiler plugin with wasi-sdk, it works fine without special action, please refer to: https://clang.llvm.org/docs/ClangPlugins.html